README.md

```diff
@@ -18,6 +18,10 @@ We are currently using [Kubeflow Pipelines 1.8.4](https://github.com/kubeflow/pi
 [Tekton >= 0.47.1](https://github.com/tektoncd/pipeline/releases/tag/v0.47.1)
 for this project.
 
+For [Kubeflow Pipelines 2.0.0](https://github.com/kubeflow/pipelines/releases/tag/2.0.0) and
+[Tekton >= 0.47.1](https://github.com/tektoncd/pipeline/releases/tag/v0.47.1)
```

Comment:
the v2-integration branch is using 0.47.3

Reply:
thanks changed it.